pmultiqc/modules/quantms/quantms.py

```
@@ -1838,6 +2015,261 @@ def getSpectrumCountAcrossRep(condition_count_dict: dict):
+        mzid_table["pep_to_prot_unique"] = mzid_table.groupby(["spectrumID", "PeptideSequence"])["accession"].transform(lambda x: True if len(set(x)) == 1 else False)
+        mzid_table["accession_group"] = mzid_table.groupby(["spectrumID", "PeptideSequence"])["accession"].transform(lambda x: ";".join(x.unique()))
+
+        if "isDecoy" not in mzid_table.columns:
+            mzid_table["isDecoy"] = False
+
+        mzid_table['filename'] = mzid_table.apply(lambda x: self.file_prefix(x.location), axis=1)
+        self.ms_with_psm = mzid_table['filename'].unique().tolist()
+
+        # TODO remove_decoy
+        if config.kwargs['remove_decoy']:
+            mzid_table = mzid_table[mzid_table["isDecoy"] == False]
```

Ruff

Avoid equality comparisons to `False`; use `if not mzid_table["isDecoy"]:` for false checks

Replace with `not mzid_table["isDecoy"]`

(E712)